sklearn/preprocessing/tests/test_data.py

```diff
@@ -851,6 +854,298 @@ def test_robust_scaler_iris_quantiles():
     assert_array_almost_equal(q_range, 1)
 
 
+def test_quantile_transform_iris():
+    X = iris.data
+    transformer = QuantileTransformer(n_quantiles=30)
+    X_trans = transformer.fit_transform(X)
+    assert_array_almost_equal(np.min(X_trans, axis=0), 0.)
```

Comment:
This seems an unnecessarily weak test when we can check that transforming some more arbitrary quantile of the input results in the expected quantile.